ui/components/app/edit-gas-fee-popover/network-statistics/network-statistics.js

```diff
@@ -28,26 +37,41 @@ const NetworkStatistics = () => {
         <I18nValue messageKey="networkStatus" />
       </Typography>
       <div className="network-statistics__info">
-        <div className="network-statistics__info__field">
-          <BaseFeeTooltip>
-            <span className="network-statistics__info__field-data">
-              {gasFeeEstimates?.estimatedBaseFee &&
-                `${roundToDecimalPlacesRemovingExtraZeroes(
-                  gasFeeEstimates?.estimatedBaseFee,
-                  0,
-                )} GWEI`}
-            </span>
-            <span className="network-statistics__info__field-label">
-              <I18nValue messageKey="baseFee" />
-            </span>
-          </BaseFeeTooltip>
-        </div>
-        <div className="network-statistics__info__separator" />
-        <LatestPriorityFeeField />
-        <div className="network-statistics__info__separator" />
-        <div className="network-statistics__info__field">
-          <StatusSlider />
-        </div>
+        {isNullish(formattedLatestBaseFee) ? null : (
+          <div
+            className="network-statistics__field"
+            data-testid="formatted-latest-base-fee"
+          >
+            <BaseFeeTooltip>
+              <span className="network-statistics__field-data">
+                {formattedLatestBaseFee}
+              </span>
+              <span className="network-statistics__field-label">
+                <I18nValue messageKey="baseFee" />
```

Comment:
We might have to remove the component from here as well [#13235](https://github.com/MetaMask/metamask-extension/issues/13235)

Reply:
Removed in b4529ae8291611dccc5594f401902bb990643c21.